tests/models/language/generation/test_hybrid.py

```diff
@@ -28,7 +30,7 @@
 ]
 
 HYBRID_MODELS = [
-    "ai21labs/Jamba-tiny-dev",
+    "ai21labs/AI21-Jamba-Reasoning-3B",
```

Comment:
3B seems on the large side for CI - is there any smaller model we could use for the test?

Reply:
Agreed, why can we not continue to use the tiny model?